### DISCUSSION TITLE
Related form

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#6355] [Help]

**Comment by melohagan** [comment]
These may be useful:

- https://docs.budibase.com/docs/form-steps
- https://docs.budibase.com/docs/referencing#example-creating-and-linking-two-table-entries-in-one-form

Otherwise I'll need a specific issue that you've encountered. 

You've started 12 discussions in the past two weeks. It may be worth digging into the Budibase platform a bit further before trying to build a fully fledged app.

Are there any high-level tutorials / types of apps you would find useful?

There's also some templates you can explore here: https://budibase.com/templates/
